explorer/src/components/SEO.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import { Helmet, HelmetProps } from 'react-helmet'
 import { useStaticQuery, graphql } from 'gatsby'
+import Suisse from '../fonts/SuisseBPIntlBold.woff2'
```

Comment:
Fonts should be added in `plugins/gatsby-plugin-top-layout/TopLayout.js`, not here.